tests/test_website.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+import aiohttp.web
+import logging
+
+from collections import namedtuple
+from pyslackersweb.website import tasks
+
+SlackInviteTestParam = namedtuple("Param", "response data expected")
+
+
+@pytest.fixture
+async def website_client(app: aiohttp.web.Application, aiohttp_client, slack_client, redis):
```

Comment:
To avoid confusion with the routing (as in L22), why not make a base `test_client` for the top-level app, so all the HTTP calls are as they would look in the browser. We can still have the `website_client` customize the `slack_client` it uses for testing.

Reply:
I don't understand what you mean.
L22 check that `/` is correctly redirect to `/web` as it is in production.

We only have one subapp for the moment so we could have one `app` fixture doing everything but once we have multiple subapps we may need to split the fixture anyways.

Reply:
The confusion I mean is that it's odd that the "website app client" can `GET /` since that is a route on the main "app" and from my view it shouldn't be reachable. 

I agree with breaking them up, I would expect us to have a `app_client`, `web_client` and `*_client` (sirbot or whatever else we add). 

But I can refactor some ideas after we merge it in. I typically namespace the tests to match the packages under test.